xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -282,19 +283,12 @@ enum MemoEditorDetailNotification: Hashable {
     /// Request detail from any audience scope
     case requestFindLinkDetail(EntryLink)
     
+    case requestSaveEntry(MemoEntry)
     case requestDelete(Slashlink?)
-    
     case requestMoveEntry(from: Slashlink, to: Slashlink)
-    case succeedMoveEntry(from: Slashlink, to: Slashlink)
```

Comment:
None of these success notifications were in use anymore.

EditorDetailNotifications only flow up the tree, but these success events flow down the tree from app since my refactor.